oop/homework.py

```diff
@@ -37,33 +37,63 @@ class Cat:
     * Implement get_average_speed and return average_speed
 
     """
+    saturation_level = 50
 
     def __init__(self, age):
-        pass
+        self.age = age
+        self.average_speed = self._set_average_speed()
 
-    def eat(self, product):
-        pass
+    def _reduce_saturation_level(self, level):
+        self.saturation_level = self.saturation_level - level
+        if self.saturation_level < 0:
+            self.saturation_level = 0
 
-    def _reduce_saturation_level(self, value):
-        pass
+    def _increase_saturation_level(self, level):
+        self.saturation_level = self.saturation_level + level
+        if self.saturation_level > 100:
+            self.saturation_level = 100
 
-    def _increase_saturation_level(self, value):
-        pass
+    def eat(self, product):
+        if product == 'fodder':
+            self._increase_saturation_level(10)
+        elif product == 'apple':
+            self._increase_saturation_level(5)
+        elif product == 'milk':
+            self._increase_saturation_level(2)
```

Comment:
to avoid multiple conditions create dict like `{'milk': 2, ...}` and in this method just call `self._increase_saturation_level(dict.get(product, 0))`